pkg/storage/storage.go

```diff
@@ -152,14 +157,14 @@ func List(client *occlient.Client, componentName string, applicationName string)
 
 	pvcs, err := client.GetPVCsFromSelector(storagelabels.StorageLabel)
 	if err != nil {
-		return nil, errors.Wrapf(err, "unable to get Deployment Config associated with component %v", componentName)
+		return nil, errors.Wrapf(err, "unable to get PVC using selector %v", storagelabels.StorageLabel)
 	}
 	var storageList []StorageInfo
 	for _, pvc := range pvcs {
 		pvcComponentName, ok := pvc.Labels[componentlabels.ComponentLabel]
 		pvcAppName, okApp := pvc.Labels[applabels.ApplicationLabel]
-		// first check if storage label does not exists indicating that the storage is not mounted in any component
-		// if the storage label exists, then check if the component is the current active component
+		// first check if component label does not exists indicating that the storage is not mounted in any component
```

Comment:
wasn't it correct before? It is checking labels on storage not on component

Reply:
it is checking the component label on storage so changed that way @kadel